### PR TITLE
Delete a lot of unused VCF constants

### LINKED ISSUE #5678: Is the Downsampled annotation still a real thing?
This `DS` annotation has been kicking around in the header for a long time, but I've never seen it in the wild.  It doesn't show up in any of our integration tests.  We only have the ability to add this annotation if the GenotypingEngine gets a non-null `Map<String, AlignmentContext> stratifiedContexts`, but that doesn't seem to be the case in *_any_* of our tests.  Maybe it's a holdover from UnifiedGenotyper?

@davidbenjamin have you seen cases where we `calculateGenotypes` with stratifiedContexts (or refContext or rawContext or likelihoods)?  Given that there's zero test coverage, how would you feel about ripping it out and seeing if anyone complains?

### PR DESCRIPTION
**Pull request by davidbenjamin**
Closes #5678.

@ldgauthier I was addressing #5678 and decided to continue cleaning house.  Do you object to any of these?  They are all unused.

### CODE REVIEW

File: src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVCFConstants.java [hunk 1]
```diff
@@ -26,60 +26,42 @@
...
-    public static final String BASE_COUNTS_KEY =                    "BaseCounts";
     public static final String BASE_QUAL_RANK_SUM_KEY =             "BaseQRankSum";
     public static final String AS_BASE_QUAL_RANK_SUM_KEY =          "AS_BaseQRankSum";
     public static final String AS_RAW_BASE_QUAL_RANK_SUM_KEY =      "AS_RAW_BaseQRankSum";
     public static final String GENOTYPE_AND_VALIDATE_STATUS_KEY =   "callStatus";
     public static final String CLIPPING_RANK_SUM_KEY =              "ClippingRankSum";
     public static final String CULPRIT_KEY =                        "culprit";
-    public static final String SPANNING_DELETIONS_KEY =             "Dels";
     public static final String ORIGINAL_DP_KEY =                    "DP_Orig"; //SelectVariants
-    public static final String DOWNSAMPLED_KEY =                    "DS";
```
Comment (ldgauthier) [inline src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVCFConstants.java]: I have a branch where I hook this up so it actually works.  It's pretty handy for debugging some types of things.  If you want to delete it now in the case that I never merge that branch, that's fair.

File: src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVCFConstants.java [hunk 2]
```diff
@@ -26,60 +26,42 @@
     public static final String AS_RAW_RMS_MAPPING_QUALITY_KEY =     "AS_RAW_MQ";
     public static final String AS_CULPRIT_KEY =                     "AS_culprit";
     public static final String AS_VQS_LOD_KEY =                     "AS_VQSLOD";
-    public static final String ALLELE_BALANCE_HET_KEY =             "ABHet";
-    public static final String ALLELE_BALANCE_HOM_KEY =             "ABHom";
```
Comment (ldgauthier) [inline src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVCFConstants.java]: These would be useful, but there's no code to generate them.  You can delete them, but they're not bad annotations.

File: src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVCFConstants.java [hunk 3]
```diff
@@ -26,60 +26,42 @@
...
     public static final String FISHER_STRAND_KEY =                  "FS";
     public static final String AS_FISHER_STRAND_KEY =               "AS_FS";
     public static final String AS_SB_TABLE_KEY =                    "AS_SB_TABLE";
     public static final String SB_TABLE_KEY =                       "SB_TABLE";
-    public static final String GC_CONTENT_KEY =                     "GC";
     public static final String GQ_MEAN_KEY =                        "GQ_MEAN";
     public static final String GQ_STDEV_KEY =                       "GQ_STDDEV";
     public static final String HAPLOTYPE_SCORE_KEY =                "HaplotypeScore";
     public static final String HI_CONF_DENOVO_KEY =                 "hiConfDeNovo";
-    public static final String HOMOPOLYMER_RUN_KEY =                "HRun";
```
Comment (ldgauthier) [inline src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVCFConstants.java]: The TandemRepeat annotation will annotate repeat unit length one, right?

File: src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVCFConstants.java [hunk 4]
```diff
@@ -26,60 +26,42 @@
...
-    public static final String LOW_MQ_KEY =                         "LowMQ";
     public static final String MLE_ALLELE_COUNT_KEY =               "MLEAC";
     public static final String MLE_ALLELE_FREQUENCY_KEY =           "MLEAF";
-    public static final String MLE_PER_SAMPLE_ALLELE_COUNT_KEY =    "MLPSAC";
-    public static final String MLE_PER_SAMPLE_ALLELE_FRACTION_KEY = "MLPSAF";
     public static final String MAP_QUAL_RANK_SUM_KEY =              "MQRankSum";
     public static final String RAW_MAP_QUAL_RANK_SUM_KEY =          "RAW_MQRankSum";
     public static final String AS_MAP_QUAL_RANK_SUM_KEY =           "AS_MQRankSum";
     public static final String AS_RAW_MAP_QUAL_RANK_SUM_KEY =       "AS_RAW_MQRankSum";
-    public static final String MENDEL_VIOLATION_LR_KEY =            "MVLR";
```
Comment (ldgauthier) [inline src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVCFConstants.java]: Was this from PhaseByTransmission?  Alas, even after the entire duration of Laurent's postdoc, we still never got that code into GATK4.

File: src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVCFConstants.java (hunk 1 above)
Comment (davidbenjamin) [reply src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVCFConstants.java]: I'll leave this in.

File: src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVCFConstants.java (hunk 3 above)
Comment (davidbenjamin) [reply src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVCFConstants.java]: Yes, it does, so this is safe.

File: src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVCFConstants.java (hunk 4 above)
Comment (davidbenjamin) [reply src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVCFConstants.java]: No idea.

**Pull request merged**